paypal/paypal-messages-android
# refactor: update PayPalEnvironment to use sealed class

## Description

Updates PayPalEnvironment to use a sealed class for better consistency

## Testing instructions

1. Run messages through a few standard render tests. There should be no change to past behavior
